### DISCUSSION TITLE
Is this just me or the Shuffle is Broken?

### DISCUSSION THREAD

**Discussion by ahmmedrejowan** [fast4x/RiMusic#5080] [Q&A]
I found a weird situation. I don't if it's a bug or not, so asking here first before reporting as issue. So the thing is, the shuffle feature works beyond its limit. 

Think of this scenario - 
You are playing from a playlist with shuffle. The logical thing would be the songs to be shuffled in the queue only from that Playlist. 
But in my case the shuffle is going beyond the current playlist and adding songs from other playlists which I don't want. 

Do any of you faced this?

**Comment by fast4x** [comment]
try latest update
